src/main/java/core/basesyntax/service/report/ReportServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.report;
+
+import core.basesyntax.dao.Storage;
+import core.basesyntax.model.FruitRecord;
+import java.util.List;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String FRUIT = "fruit";
+    private static final String QUANTITY = "quantity";
+    private static final String COMMA = ",";
+
+    @Override
+    public String getReport(List<FruitRecord> fruitRecords) {
+        StringBuilder report = new StringBuilder();
+        report.append(FRUIT).append(COMMA).append(QUANTITY).append(System.lineSeparator());
+
+        Storage.storage.forEach((fruit, quantity) -> report
```

Comment:
Avoid using static access to the database storage directly. It would be better to pass the required data as a parameter to the method, which aligns with the principle of making services independent as stated in the checklist.